src/Admin/ajv.ts

```diff
@@ -196,6 +197,29 @@ function transformWordPressSchemaToDraft7(schema: JSONSchemaType<any>): JSONSche
                 delete prop.required;
             }
 
+            // Handle WordPress schema with Array type and enum
+            if (prop && typeof prop === 'object' && !Array.isArray(prop)) {
```

Comment:
I noticed we're doing this check for `prop && typeof prop === 'object'` for every conditional.  Perhaps we can clean this up and just return early if this is false so the rest are clear to checking for specific things.

Reply:
Much better, thanks!